File: Data Structures & Algorithms/two-integer-sum/submission-3.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        for(var i = 0; i < nums.length; i++){
            for(var j = i + 1; j < nums.length; j++){
                var total = nums[i] + nums[j];
                if(total === target){
                    return [i, j];
                }
            }
        }
        return [];
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-3.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l, r = 0, len(numbers) - 1

        # lets move the points from either end and add l + r to check if its equal to target

        while l < r:
            sum = numbers[l] + numbers[r]

            if sum < target:
                l += 1
            elif sum > target:
                r -= 1
            else:
                return [l+1,r+1]
        return []